extensions/toggl-track/src/components/UpdateTimeEntryForm.tsx

```diff
@@ -33,6 +33,8 @@ function UpdateTimeEntryForm({ timeEntry, revalidateTimeEntries }: UpdateTimeEnt
   const [billable, setBillable] = useState(timeEntry.billable);
 
   const [taskSearch, setTaskSearch] = useState("");
+  const [startDate, setStartDate] = useState<Date | null>(timeEntry.start ? new Date(timeEntry.start) : null);
```

Comment:
You could store `const defaultStartDate = timeEntry.start ? new Date(timeEntry.start) : null` and use this both in the useState, and in the defaultValues in form. Minor change, but a bit cleaner

Reply:
Same goes for end